MIGRATION.md

```diff
@@ -3,6 +3,13 @@
 The main goal of version 7 is to reduce bundle size. This version is breaking because we removed deprecated APIs, upgraded our build tooling, and restructured npm package contents.
 Below we will outline all the breaking changes you should consider when upgrading.
 
+TL;DR if you copy & pasted the basic examples from our docs:
```

Comment:
```suggestion
TL;DR: If you just use basic features of Sentry (e.g. you copy & pasted the basic examples from our docs), here's what changed for you:
```

Reply:
Totally optional but maybe something along those lines to make the intent of the TL;DR a bit clearer. WDYT?

Reply:
Slightly adjusted because i didn't like the parenthesis. Let me know what you think of it now!

Reply:
Nice, I like it. Thanks for taking another look!